garnett/drf.py

```diff
@@ -0,0 +1,69 @@
+"""
+Helper methods for working with django-rest-framework
+"""
+
+from enum import Enum
+from rest_framework.fields import JSONField, empty
+from rest_framework import serializers
+
+from garnett.fields import TranslatedField
+
+
+class LanguageTypes(Enum):
+    monolingual = 1
+    multilingual = 2
+
+
+class TranslatableAPIField(JSONField):
+    def __init__(self, *args, **kwargs):
+        self.innerfield = kwargs.pop("innerfield", None)
+        super().__init__(*args, **kwargs)
+
+    @property
+    def validators(self):
+        if not hasattr(self, "_validators"):
+            self._validators = self.get_validators()
+
+        if self.translation_type == LanguageTypes.monolingual:
+            return self.innerfield.validators
+
+        return self._validators
+
+    @validators.setter
+    def validators(self, validators):
+        self._validators = validators
+
+    def run_validation(self, data=empty):
+        (is_empty_value, data) = self.validate_empty_values(data)

```

Comment:
Brackets aren't necessary, but also I don't think they are particularly harmful

Reply:
This was lifted straight from django-rest-framework